fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/util/CalendarHelper.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * (C) Copyright IBM Corp. 2021
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+ 
+package com.ibm.fhir.persistence.jdbc.util;
+
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Timestamp;
+import java.sql.Types;
+import java.util.Calendar;
+import java.util.TimeZone;
+
+/**
+ * Helper for time and calendar-related functions related to JDBC statements and results
+ */
+public class CalendarHelper {
+    // Calendar is not thread-safe, but is non-trivial to construct, so we hold in thread-local
+    private static final ThreadLocal<Calendar> UTC_CALENDAR = new ThreadLocal<Calendar>() {
+        protected Calendar initialValue() { 
+            return Calendar.getInstance(TimeZone.getTimeZone("UTC")); 
+        };
+    };
+    
+    /**
+     * Get a thread-specific instance of a calendar configured for UTC.
+     * @return
+     */
+    public static Calendar getCalendarForUTC() {
+        return UTC_CALENDAR.get();
+    }
+    
+    /**
+     * Helper to obtain a UTC timestamp from a JDBC {@link ResultSet}
+     * @param rs
+     * @param col
+     * @return
+     * @throws SQLException
+     */
+    public static Timestamp getTimestampUTC(ResultSet rs, int col) throws SQLException {
```

Comment:
Are these getTimestampUTC and setTimestampUTC methods just here for future use? If so, that's fine, I just didn't see them used yet.

Reply:
Yes, I put them in planning to use them, but wanted to avoid making bigger changes than necessary at this late stage before 4.9.1.